src/components/Disputes/DisputeCard.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import {
+  Card,
+  GU,
+  ContextMenu,
+  ContextMenuItem,
+  IconInfo,
+  IdentityBadge,
+  textStyle,
+  useTheme,
+} from '@aragon/ui'
+
+import DisputeText from './DisputeText'
+import DisputeStatus from './DisputeStatus'
+
+function DisputeCard({ dispute, selectDispute }) {
+  const theme = useTheme()
+
+  return (
+    <CardItem>
+      <div
+        css={`
+          display: flex;
+          align-items: center;
+          justify-content: space-between;
+        `}
+      >
+        <DisputeStatus dispute={dispute} />
+        <ContextMenu>
+          <ContextMenuItem onClick={() => selectDispute(dispute.id)}>
+            <IconInfo />
+            View
+          </ContextMenuItem>
+        </ContextMenu>
+      </div>
+
+      <div
+        css={`
+          & > * {
+            margin-bottom: ${1 * GU}px;
+          }
+        `}
+      >
+        <h3
+          css={`
+            ${textStyle('body1')}
+          `}
+        >
+          Dispute
+          <strong> #{dispute.id}</strong>
```

Comment:
should we define those attributes as const {id, description, rewardAmount, stakedAmount} = dispute?